src/integrations/utils.js

```diff
@@ -18,16 +19,23 @@ export function dynamicImport(dependency, async = true, index = 0) {
   }
 }
 
-export async function initializeIntegrations(integrations, clientArg, init) {
+export async function initializeIntegrations(integrations, clientArg) {
+  // This fixes the test that was failing, but it's not really clear to me why this started failing
```

Comment:
which test?

Reply:
hmm, I removed it and the tests all pass